libraries/bezier-rs/src/subpath/core.rs

```diff
@@ -34,6 +34,47 @@ impl<ManipulatorGroupId: crate::Identifier> Subpath<ManipulatorGroupId> {
 		)
 	}
 
+	/// Creates a subpath from a vector of [Bezier]. When two consecutive Bezier do not share an end and start point, this function
```

Comment:
"vector of [Bezier]" -> "slice of [Bezier]"

"two consecutive Bezier" -> "two consecutive Beziers"